reciprocalspaceship/utils/structurefactors.py

```diff
@@ -75,16 +77,7 @@ def compute_structurefactor_multiplicity(H, sg, include_centering=True):
         an array of length n containing the multiplicity
         of each hkl.
     """
-    if isinstance(sg, SpaceGroup):
-        group_ops = sg.operations()
-    elif isinstance(sg, GroupOps):
-        group_ops = sg
-    else:
-        raise ValueError(
-            f"gemmi.SpaceGroup or gemmi.GroupOps expected for parameter sg. "
-            f"Received object of type: ({type(sg)}) instead."
-        )
-
```

Comment:
Note: This change removes support for invoking this function directly with a `GroupOps` object. We do not actually use this anywhere in our codebase, so it seems like the added flexibility of `@spacegroupify` is preferable. This is also true for `is_absent()` and `is_centric()`.